<!DOCTYPE HTML>
<html>
    <head>
        <!-- Global site tag (gtag.js) - Google Analytics -->
        <script async src="https://www.googletagmanager.com/gtag/js?id=UA-89797207-1"></script>
        <script>
          window.dataLayer = window.dataLayer || [];
          function gtag(){dataLayer.push(arguments);}
          gtag('js', new Date());

          gtag('config', 'UA-89797207-1');
        </script>
        <link rel="shortcut icon" type="image/x-icon" href="images/favicon.ico">
        <title>Min Woo Sun</title>
        <meta charset="utf-8" />
        <meta name="viewport" content="width=1000">
        <link rel="stylesheet" href="https://use.typekit.net/quv7bsd.css"> <!-- fonts -->
        <link rel="stylesheet" href="index.css" />
    </head>
    <body id="body">
        <div class="topnav">
          <a href="/">Home</a>
          <a class="active" href="research">Research</a>
          <a href="talks">Talks</a>
          <a href="experience">Experience</a>         
          <a href="education">Education</a>
        </div>
        
        <div id="main"> 
            <header id="header">
            </header>
 
               <div class="section research">
                <h1>Publications</h1>
                   
                <div class="research-proj">
                    <a href="" class="research-thumb"><img src="" alt="" /></a>
                    <div class="research-proj-desc">
                        <h2><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC9034430/">Improved Digital Therapy for Developmental Pediatrics Using Domain-Specific Artificial Intelligence: Machine Learning Study</a></h2>
                        <p>
                            Peter Washington, Haik Kalantarian, John Kent, Arman Husic, Aaron Kline, Emilie Leblanc, Cathy Hou, Onur Cezmi Mutlu, Kaitlyn Dunlap, Yordan Penev, Maya Varma, Nate Stockham, Brianna Chrisman, Kelley Paskov, Min Woo Sun, Jae-Yoon Jung, Catalin Voss, Nick Haber, Dennis P. Wall
                            <br>
                            <i>JMIR Pediatrics and Parenting 2022</i><br>
                                <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC9034430/pdf/pediatrics_v5i2e26760.pdf">PDF</a>
                        </p>
                        <p>
                            Background: Automated emotion classification could aid those who struggle to recognize emotions, including children with developmental behavioral conditions such as autism. However, most computer vision emotion recognition models are trained on adult emotion and therefore underperform when applied to child faces.Objective: We designed a strategy to gamify the collection and labeling of child emotion–enriched images to boost the performance of automatic child emotion recognition models to a level closer to what will be needed for digital health care approaches. Methods: We leveraged our prototype therapeutic smartphone game, GuessWhat, which was designed in large part for children with developmental and behavioral conditions, to gamify the secure collection of video data of children expressing a variety of emotions prompted by the game. Independently, we created a secure web interface to gamify the human labeling effort, called <a href="javascript:toggleBio('pub9')">[...]</a> 
                            <div id="pub9" style="display: None">
                                <p>HollywoodSquares, tailored for use by any qualified labeler. We gathered and labeled 2155 videos, 39,968 emotion frames, and 106,001 labels on all images. With this drastically expanded pediatric emotion–centric database (>30 times larger than existing public pediatric emotion data sets), we trained a convolutional neural network (CNN) computer vision classifier of happy, sad, surprised, fearful, angry, disgust, and neutral expressions evoked by children. Results: The classifier achieved a 66.9% balanced accuracy and 67.4% F1-score on the entirety of the Child Affective Facial Expression (CAFE) as well as a 79.1% balanced accuracy and 78% F1-score on CAFE Subset A, a subset containing at least 60% human agreement on emotions labels. This performance is at least 10% higher than all previously developed classifiers evaluated against CAFE, the best of which reached a 56% balanced accuracy even when combining “anger” and “disgust” into a single class. Conclusions: This work validates that mobile games designed for pediatric therapies can generate high volumes of domain-relevant data sets to train state-of-the-art classifiers to perform tasks helpful to precision health efforts.</p>
                            </div>
                        
                        </p>
                    </div>
                </div>
            
                <div class="research-proj">
                    <a href="" class="research-thumb"><img src="" alt="" /></a>
                    <div class="research-proj-desc">
                        <h2><a href="https://biodatamining.biomedcentral.com/articles/10.1186/s13040-021-00262-x">A Maximum Flow-based Network Approach for Identification of Stable Noncoding Biomarkers Associated with the Multigenic Neurological Condition, Autism</a></h2>
                        <p>
                            Maya Varma, Kelley Paskov, Brianna Chrisman, Min Woo Sun, Jae-Yoon Jung, Nate Stockham, Peter Washington, Dennis P. Wall
                            <br>
                            <i>BMC BioData Mining 2021</i><br>
                                <a href="https://biodatamining.biomedcentral.com/track/pdf/10.1186/s13040-021-00262-x.pdf">PDF</a>&nbsp;&nbsp;&bull;&nbsp;&nbsp;
                                <a href="https://github.com/maya124/MaximumFlow">Code</a>
                        </p>
                        <p>
                            Background: Machine learning approaches for predicting disease risk from high-dimensional whole genome sequence (WGS) data often result in unstable models that can be difficult to interpret, limiting the identification of putative sets of biomarkers. Here, we design and validate a graph-based methodology based on maximum flow, which leverages the presence of linkage disequilibrium (LD) to identify stable sets of variants associated with complex multigenic disorders. Results: We apply our method to a previously published logistic regression model trained to identify variants in simple repeat sequences associated with autism spectrum disorder (ASD); this L1-regularized model exhibits high predictive accuracy yet demonstrates great variability in the features selected from over 230,000 possible variants. In order to improve model stability, we extract the variants assigned non-zero weights in each of 5 cross-validation folds and then assemble the five sets of features <a href="javascript:toggleBio('pub8')">[...]</a> 
                            <div id="pub8" style="display: None">
                                <p>into a flow network subject to LD constraints. The maximum flow formulation allowed us to identify 55 variants, which we show to be more stable than the features identified by the original classifier. Conclusion: Our method allows for the creation of machine learning models that can identify predictive variants. Our results help pave the way towards biomarker-based diagnosis methods for complex genetic disorders.</p>
                            </div>
                        
                        </p>
                    </div>
                </div>

                <div class="research-proj">
                    <a href="" class="research-thumb"><img src="" alt="" /></a>
                    <div class="research-proj-desc">
                        <h2><a href="https://biodatamining.biomedcentral.com/articles/10.1186/s13040-021-00259-6">Estimating Sequencing Error Rates Using Families</a></h2>
                        <p>
                            Kelley Paskov, Jae-Yoon Jung, Brianna Chrisman, Nate T. Stockham, Peter Washington, Maya Varma, Min Woo Sun, Dennis P. Wall
                            <br>
                            <i>BMC BioData Mining 2021</i><br>
                                <a href="https://biodatamining.biomedcentral.com/track/pdf/10.1186/s13040-021-00259-6.pdf">PDF</a>&nbsp;&nbsp;&bull;&nbsp;&nbsp;
                                <a href="https://github.com/kpaskov/FamilySeqError">Code</a>&nbsp;&nbsp;&bull;&nbsp;&nbsp;
                                <a href="http://www.ihart.org/home">iHART dataset</a>
                        </p>
                        <p>
                            As next-generation sequencing technologies make their way into the clinic, knowledge of their error rates is essential if they are to be used to guide patient care. However, sequencing platforms and variant-calling pipelines are continuously evolving, making it difficult to accurately quantify error rates for the particular combination of assay and software parameters used on each sample. Family data provide a unique opportunity for estimating sequencing error rates since it allows us to observe a fraction of sequencing errors as Mendelian errors in the family, which we can then use to produce genome-wide error estimates for each sample. Results: We introduce a method that uses Mendelian errors in sequencing data to make highly granular per-sample <a href="javascript:toggleBio('pub7')">[...]</a>
                            <div id="pub7" style="display: None">
                                <p>estimates of precision and recall for any set of variant calls, regardless of sequencing platform or calling methodology. We validate the accuracy of our estimates using monozygotic twins, and we use a set of monozygotic quadruplets to show that our predictions closely match the consensus method. We demonstrate our method’s versatility by estimating sequencing error rates for whole genome sequencing, whole exome sequencing, and microarray datasets, and we highlight its sensitivity by quantifying performance increases between different versions of the GATK variant-calling pipeline. We then use our method to demonstrate that: 1) Sequencing error rates between samples in the same dataset can vary by over an order of magnitude. 2) Variant calling performance decreases substantially in low-complexity regions of the genome. 3) Variant calling performance in whole exome sequencing data decreases with distance from the nearest target region. 4) Variant calls from lymphoblastoid cell lines can be as accurate as those from whole blood. 5) Whole-genome sequencing can attain microarray-level precision and recall at disease-associated SNV sites. Conclusion: Genotype datasets from families are powerful resources that can be used to make fine-grained estimates of sequencing error for any sequencing platform and variant-calling methodology.</p>
                            </div>
                        
                        </p>
                    </div>
                </div>
        
                <div class="research-proj">
                    <a href="" class="research-thumb"><img src="" alt="" /></a>
                    <div class="research-proj-desc">
                        <h2><a href="https://biodatamining.biomedcentral.com/articles/10.1186/s13040-021-00251-0">Indels in SARS-CoV-2 Occur at Template-switching Hotspots</a></h2>
                        <p>
                            Brianna Chrisman, Kelley Paskov, Nate Stockham, Kevin Tabatabaei, Jae-Yoon Jung, Peter Washington, Maya Varma, Min Woo Sun, Sepideh Maleki, Dennis P. Wall
                            <br>
                            <i>BMC BioData Mining 2021</i><br>
                                <a href="https://biodatamining.biomedcentral.com/track/pdf/10.1186/s13040-021-00251-0.pdf">PDF</a>&nbsp;&nbsp;&bull;&nbsp;&nbsp;
                                <a href="https://github.com/briannachrisman/sars-cov2-SVs">Code</a>
                        </p>
                        <p>
                            The evolutionary dynamics of SARS-CoV-2 have been carefully monitored since the COVID-19 pandemic began in December 2019. However, analysis has focused primarily on single nucleotide polymorphisms and largely ignored the role of insertions and deletions (indels) as well as recombination in SARS-CoV-2 evolution. Using sequences from the GISAID database, we catalogue over 100 insertions and deletions in the SARS-CoV-2 consensus sequences. We hypothesize that these indels are artifacts of recombination events between SARS-CoV-2 replicates whereby RNA-dependent RNA polymerase (RdRp) re-associates with a homologous template at a different loci (“imperfect homologous recombination”). We provide several independent pieces of <a href="javascript:toggleBio('pub6')">[...]</a>
                            <div id="pub6" style="display: None">
                                <p>evidence that suggest this. (1) The indels from the GISAID consensus sequences are clustered at specific regions of the genome. (2) These regions are also enriched for 5’ and 3’ breakpoints in the transcription regulatory site (TRS) independent transcriptome, presumably sites of RNA-dependent RNA polymerase (RdRp) template-switching. (3) Within raw reads, these indel hotspots have cases of both high intra-host heterogeneity and intra-host homogeneity, suggesting that these indels are both consequences of de novo recombination events within a host and artifacts of previous recombination. We briefly analyze the indels in the context of RNA secondary structure, noting that indels preferentially occur in “arms” and loop structures of the predicted folded RNA, suggesting that secondary structure may be a mechanism for TRS-independent template-switching in SARS-CoV-2 or other coronaviruses. These insights into the relationship between structural variation and recombination in SARS-CoV-2 can improve our reconstructions of the SARS-CoV-2 evolutionary history as well as our understanding of the process of RdRp template-switching in RNA viruses.</p>
                            </div>
                        
                        </p>
                    </div>
                </div>

                <div class="research-proj">
                    <a href="" class="research-thumb"><img src="" alt="" /></a>
                    <div class="research-proj-desc">
                        <h2><a href="https://bmcbioinformatics.biomedcentral.com/articles/10.1186/s12859-020-03693-1">Game Theoretic Centrality: A Novel Approach to Prioritize Disease Candidate Genes by Combining Biological Networks with the Shapley Value</a></h2>
                        <p>
                            Min Woo Sun, Stefano Moretti, Kelley Paskov, Nate Stockham, Maya Varma, Brianna Chrisma, Peter Washington, Jae-Yoon Jung, Dennis P. Wall
                            <br>
                            <i>BMC Bioinformatics 2020</i><br>
                                <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-020-03693-1.pdf">PDF</a>&nbsp;&nbsp;&bull;&nbsp;&nbsp;
                                <a href="https://github.com/minwoosun">Code</a>&nbsp;&nbsp;&bull;&nbsp;&nbsp;
                                <a href="http://www.ihart.org/home">iHART dataset</a>
                        </p>
                        <p>
                            Background: Complex human health conditions with etiological heterogeneity like Autism Spectrum Disorder (ASD) often pose a challenge for traditional genome-wide association study approaches in defining a clear genotype to phenotype model. Coalitional game theory (CGT) is an exciting method that can consider the combinatorial effect of groups of variants working in concert to produce a phenotype. CGT has been applied to associate likely-gene-disrupting variants encoded from whole genome sequence data to ASD; however, this previous approach cannot take into account for prior biological knowledge. Here we extend CGT to incorporate a priori knowledge from biological networks through a game theoretic centrality measure based on Shapley value to rank genes by their relevance–the individual gene’s synergistic influence in a gene-to-gene interaction network. Game theoretic centrality extends the notion of Shapley value to the evaluation of a gene’s contribution to the overall connectivity of its corresponding node in a biological network. Results: We <a href="javascript:toggleBio('pub5')">[...]</a>
                            <div id="pub5" style="display: None">
                                <p>implemented and applied game theoretic centrality to rank genes on whole genomes from 756 multiplex autism families. Top ranking genes with the highest game theoretic centrality in both the weighted and unweighted approaches were enriched for pathways previously associated with autism, including pathways of the immune system. Four of the selected genes HLA-A, HLA-B, HLA-G, and HLA-DRB1–have also been implicated in ASD and further support the link between ASD and the human leukocyte antigen complex. Conclusions: Game theoretic centrality can prioritize influential, disease-associated genes within biological networks, and assist in the decoding of polygenic associations to complex disorders like autism.</p>
                            </div>
                        
                        </p>
                    </div>
                </div>

                <div class="research-proj">
                    <a href="" class="research-thumb"><img src="" alt="" /></a>
                    <div class="research-proj-desc">
                        <h2><a href="https://www.mdpi.com/2075-4426/10/3/86">Precision Telemedicine through Crowdsourced Machine Learning: Testing Variability of Crowd Workers for Video-Based Autism Feature Recognition</a></h2>
                        <p>
                            Peter Washington, Emilie Leblanc, Kaitlyn Dunlap, Yordan Penev, Aaron Kline, Kelley Paskov, Min Woo Sun, Brianna Chrisman, Nathaniel Stockham, Maya Varma, Catalin Voss, Nick Haber, Dennis P Wall
                            <br>
                            <i>MDPI 2020</i><br>
                                <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7564950/pdf/jpm-10-00086.pdf">PDF</a>
                        </p>
                        <p>
                            Mobilized telemedicine is becoming a key, and even necessary, facet of both precision health and precision medicine. In this study, we evaluate the capability and potential of a crowd of virtual workers—defined as vetted members of popular crowdsourcing platforms—to aid in the task of diagnosing autism. We evaluate workers when crowdsourcing the task of providing categorical ordinal behavioral ratings to unstructured public YouTube videos of children with autism and neurotypical controls. To evaluate emerging patterns that are consistent across independent crowds, we target workers from distinct geographic loci on two crowdsourcing platforms: an international group of workers on Amazon Mechanical Turk (MTurk) (N = 15) and Microworkers from <a href="javascript:toggleBio('pub4')">[...]</a>
                            <div id="pub4" style="display: None">
                                <p>Bangladesh (N = 56), Kenya (N = 23), and the Philippines (N = 25). We feed worker responses as input to a validated diagnostic machine learning classifier trained on clinician-filled electronic health records. We find that regardless of crowd platform or targeted country, workers vary in the average confidence of the correct diagnosis predicted by the classifier. The best worker responses produce a mean probability of the correct class above 80% and over one standard deviation above 50%, accuracy and variability on par with experts according to prior studies. There is a weak correlation between mean time spent on task and mean performance (r = 0.358, p = 0.005). These results demonstrate that while the crowd can produce accurate diagnoses, there are intrinsic differences in crowdworker ability to rate behavioral features. We propose a novel strategy for recruitment of crowdsourced workers to ensure high quality diagnostic evaluations of autism, and potentially many other pediatric behavioral health conditions. Our approach represents a viable step in the direction of crowd-based approaches for more scalable and affordable precision medicine.</p>
                            </div>
                        
                        </p>
                    </div>
                </div>

                <div class="research-proj">
                    <a href="" class="research-thumb"><img src="" alt="" /></a>
                    <div class="research-proj-desc">
                        <h2><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7958981/">Selection of Trustworthy Crowd Workers For Telemedical Diagnosis of Pediatric Autism Spectrum Disorder</a></h2>
                        <p>
                            Peter Washington, Emilie Leblanc, Kaitlyn Dunlap, Yordan Penev, Maya Varma, Jae-Yoon Jung, Brianna Chrisman, Min Woo Sun, Nathaniel Stockham, Kelley Paskov, Haik Kalantarian, Catalin Voss, Nick Haber, Dennis P. Wall
                            <br>
                            <i>Pacific Symposium on Biocomputing 2021</i><br>
                                <a href="https://psb.stanford.edu/psb-online/proceedings/psb21/washington.pdf">PDF</a>
                        </p>
                        <p>
                            Crowd-powered telemedicine has the potential to revolutionize healthcare, especially during times that require remote access to care. However, sharing private health data with strangers from around the world is not compatible with data privacy standards, requiring a stringent filtration process to recruit reliable and trustworthy workers who can go through the proper training and security steps. The key challenge, then, is to identify capable, trustworthy, and reliable workers through high-fidelity evaluation tasks without exposing any sensitive patient data during the evaluation process. We contribute a set of experimentally validated metrics for assessing the trustworthiness and reliability of crowd workers tasked with providing behavioral feature <a href="javascript:toggleBio('pub3')">[...]</a>
                            <div id="pub3" style="display: None">
                                <p>tags to unstructured videos of children with autism and matched neurotypical controls. The workers are blinded to diagnosis and blinded to the goal of using the features to diagnose autism. These behavioral labels are fed as input to a previously validated binary logistic regression classifier for detecting autism cases using categorical feature vectors. While the metrics do not incorporate any ground truth labels of child diagnosis, linear regression using the 3 correlative metrics as input can predict the mean probability of the correct class of each worker with a mean average error of 7.51% for performance on the same set of videos and 10.93% for performance on a distinct balanced video set with different children. These results indicate that crowd workers can be recruited for performance based largely on behavioral metrics on a crowdsourced task, enabling an affordable way to filter crowd workforces into a trustworthy and reliable diagnostic workforce.</p>
                            </div>
                        
                        </p>
                    </div>
                </div>

                <div class="research-proj">
                    <a href="" class="research-thumb"><img src="" alt="" /></a>
                    <div class="research-proj-desc">
                        <h2><a href="https://journals.sagepub.com/doi/full/10.1177/1178222619832859">Coalitional Game Theory Facilitates Identification of Non-Coding Variants Associated With Autism</a></h2>
                        <p>
                            Min Woo Sun*, Anika Gupta*, Maya Varma, Kelley Paskov, Jae-Yoon Jung, Nate Stockham, Dennis P. Wall
                            <br>
                            <i>Biomedical Informatics Insights 2019</i><br>
                                <a href="https://journals.sagepub.com/doi/pdf/10.1177/1178222619832859">PDF</a>
                        </p>
                        <p>
                            Studies on autism spectrum disorder (ASD) have amassed substantial evidence for the role of genetics in the disease’s phenotypic manifestation. A large number of coding and non-coding variants with low penetrance likely act in a combinatorial manner to explain the variable forms of ASD. However, many of these combined interactions, both additive and epistatic, remain undefined. Coalitional game theory (CGT) is an approach that seeks to identify players (individual genetic variants or genes) who tend to improve the performance—association to a disease phenotype of interest—of any coalition (subset of co-occurring genetic variants) they join. This method has been previously applied to boost biologically informative signal from gene <a href="javascript:toggleBio('pub2')">[...]</a>
                            <div id="pub2" style="display: None">
                                <p>expression data and exome sequencing data but remains to be explored in the context of cooperativity among non-coding genomic regions. We describe our extension of previous work, highlighting non-coding chromosomal regions relevant to ASD using CGT on alteration data of 4595 fully sequenced genomes from 756 multiplex families. Genomes were encoded into binary matrices for three types of non-coding regions previously implicated in ASD and separated into ASD (case) and unaffected (control) samples. A player metric, the Shapley value, enabled determination of individual variant contributions in both sets of cohorts. A total of 30 non-coding positions were found to have significantly elevated player scores and likely represent significant contributors to the genetic coordination underlying ASD. Cross-study analyses revealed that a subset of mutated non-coding regions (all of which are in human accelerated regions (HARs)) and related genes are involved in biological pathways or behavioral outcomes known to be affected in autism, suggesting the importance of single nucleotide polymorphisms (SNPs) within HARs in ASD. These findings support the use of CGT in identifying hidden yet influential non-coding players from large-scale genomic data, to better understand the precise underpinnings of complex neurodevelopmental disorders such as autism.</p>
                            </div>
                        
                        </p>
                    </div>
                </div>

                <div class="research-proj">
                    <a href="" class="research-thumb"><img src="" alt="" /></a>
                    <div class="research-proj-desc">
                        <h2><a href="https://journals.sagepub.com/doi/full/10.1177/1178222619832859">Coalitional game theory as a promising approach to identify candidate autism genes</a></h2>
                        <p>
                            Anika Gupta*, Min Woo Sun*, Kelley Paskov, Nate Stockham, Jae-Yoon Jung, Dennis P. Wall
                            <br>
                            <i>Pacific Symposium on Biocomputing 2018</i><br>
                                <a href="https://psb.stanford.edu/psb-online/proceedings/psb18/gupta.pdf">PDF</a>
                        </p>
                        <p>
                            Despite mounting evidence for the strong role of genetics in the phenotypic manifestation of Autism Spectrum Disorder (ASD), the specific genes responsible for the variable forms of ASD remain undefined. ASD may be best explained by a combinatorial genetic model with varying epistatic interactions across many small effect mutations. Coalitional or cooperative game theory is a technique that studies the combined effects of groups of players, known as coalitions, seeking to identify players who tend to improve the performance--the relationship to a specific disease phenotype--of any coalition they join. This method has been previously shown to boost biologically informative signal in gene expression data but to-date has not been applied  <a href="javascript:toggleBio('pub1')">[...]</a>
                            <div id="pub1" style="display: None">
                                <p>to the search for cooperative mutations among putative ASD genes. We describe our approach to highlight genes relevant to ASD using coalitional game theory on alteration data of 1,965 fully sequenced genomes from 756 multiplex families. Alterations were encoded into binary matrices for ASD (case) and unaffected (control) samples, indicating likely gene-disrupting, inherited mutations in altered genes. To determine individual gene contributions given an ASD phenotype, a "player" metric, referred to as the Shapley value, was calculated for each gene in the case and control cohorts. Sixty seven genes were found to have significantly elevated player scores and likely represent significant contributors to the genetic coordination underlying ASD. Using network and cross-study analysis, we found that these genes are involved in biological pathways known to be affected in the autism cases and that a subset directly interact with several genes known to have strong associations to autism. These findings suggest that coalitional game theory can be applied to large-scale genomic data to identify hidden yet influential players in complex polygenic disorders such as autism.</p>
                            </div>
                        
                        </p>
                    </div>
                </div>
                   
 <!-- publication / preprint  -->
<div class="divider"></div>                
            <div class="section research">
                <h1>Preprints /
                <br>
                Under Review
                </h1>
                <div class="research-proj">
                    <a href="" class="research-thumb"><img src="" alt="" /></a>
                    <div class="research-proj-desc">
                        <h2><a href="https://arxiv.org/abs/2201.10770">Confidence Intervals for the Cox Model Test Error from Cross-validation</a></h2>
                        <p>
                            Min Woo Sun, Robert Tibshirani
                            <br>
                            <i>arXiv 2022</i><br>
                                <a href="https://arxiv.org/pdf/2201.10770.pdf">PDF</a>&nbsp;&nbsp;&bull;&nbsp;&nbsp;
                                <a href="https://github.com/minwoosun">Code</a>&nbsp;&nbsp;&bull;&nbsp;&nbsp;
                                <a href="https://precog.stanford.edu/">PRECOG dataset</a>
                        </p>
                        <p>
                            Cross-validation (CV) is one of the most widely used techniques in statistical learning for estimating the test error of a model, but its behavior is not yet fully understood. It has been shown that standard confidence intervals for test error using estimates from CV may have coverage below nominal levels. This phenomenon occurs because each sample is used in both the training and testing procedures during CV and as a result, the CV estimates of the errors become correlated. Without accounting for this correlation, the estimate of the variance is smaller than it should be. One way to mitigate this issue is by estimating the mean squared error of the prediction error instead using nested CV. This approach has been shown to achieve superior coverage compared to intervals derived from standard CV. In this work, we generalize the nested CV idea to the Cox proportional hazards model and explore various choices of test error for this setting.
                        </p>
                    </div>
                </div>
                <div class="research-proj">
                    <a href="" class="research-thumb"><img src="" alt="" /></a>
                    <div class="research-proj-desc">
                        <h2><a href="https://www.medrxiv.org/content/10.1101/2021.06.20.21259187v1">Identification of Social Engagement Indicators Associated with Autism Spectrum Disorder Using a Game-based Mobile Application</a></h2>
                        <p>
                            Maya Varma, Peter Washington, Brianna Chrisman, Aaron Kline, Emilie Leblanc, Kelley Paskov, Nate Stockham, Jae-Yoon Jung, Min Woo Sun, Dennis P. Wall
                            <br>
                            <i>medRxiv 2021</i><br>
                                <a href="https://www.medrxiv.org/content/10.1101/2021.06.20.21259187v1.full.pdf">PDF</a>
                        </p>
                        <p>
                            Autism spectrum disorder (ASD) is a widespread neurodevelopmental condition with a range of potential causes and symptoms. Children with ASD exhibit behavioral and social impairments, giving rise to the possibility of utilizing computational techniques to evaluate a child’s social phenotype from home videos. Methods: Here, we use a mobile health application to collect over 11 hours of video footage depicting 95 children engaged in gameplay in a natural home environment. We utilize automated dataset annotations to analyze two social indicators that have previously been shown to differ between children with ASD and their neurotypical (NT) peers: (1) gaze fixation patterns and (2) visual scanning methods. We compare the gaze fixation and visual scanning methods utilized by children during a 90-second gameplay video in order to identify statistically-significant differences between the two cohorts; we then train an LSTM neural network <a href="javascript:toggleBio('prepr2')">[...]</a> 
                            <div id="prepr2" style="display: None">
                                 <p>in order to determine if gaze indicators could be predictive of ASD. Results: Our work identifies one statistically significant region of fixation and one significant gaze transition pattern that differ between our two cohorts during gameplay. In addition, our deep learning model demonstrates mild predictive power in identifying ASD based on coarse annotations of gaze fixations.Discussion: Ultimately, our results demonstrate the utility of game-based mobile health platforms in quantifying visual patterns and providing insights into ASD. We also show the importance of automated labeling techniques in generating large-scale datasets while simultaneously preserving the privacy of participants. Our approaches can generalize to other healthcare needs.</p>
                            </div>
                        </p>
                    </div>
                </div>
                <div class="research-proj">
                    <a href="" class="research-thumb"><img src="" alt="" /></a>
                    <div class="research-proj-desc">
                        <h2><a href="https://arxiv.org/abs/2012.08678">Training an Emotion Detection Classifier using Frames from a Mobile Therapeutic Game for Children with Developmental Disorders</a></h2>
                        <p>
                            Peter Washington, Haik Kalantarian, Jack Kent, Arman Husic, Aaron Kline, Emilie Leblanc, Cathy Hou, Cezmi Mutlu, Kaitlyn Dunlap, Yordan Penev, Maya Varma, Nate Stockham, Brianna Chrisman, Kelley Paskov, Min Woo Sun, Jae-Yoon Jung, Catalin Voss, Nick Haber, Dennis P. Wall
                            <br>
                            <i>arXiv 2020</i><br>
                                <a href="https://arxiv.org/pdf/2012.08678.pdf">PDF</a>
                        </p>
                        <p>
                            Automated emotion classification could aid those who struggle to recognize emotion, including children with developmental behavioral conditions such as autism. However, most computer vision emotion models are trained on adult affect and therefore underperform on child faces. In this study, we designed a strategy to gamify the collection and the labeling of child affect data in an effort to boost the performance of automatic child emotion detection to a level closer to what will be needed for translational digital healthcare. We leveraged our therapeutic smartphone game, GuessWhat, which was designed in large part for children with developmental and behavioral conditions, to gamify the secure collection of video data of children expressing a variety of emotions prompted by the game. Through a secure web interface gamifying the human labeling effort, we gathered and labeled 2,155 videos, 39,968 emotion frames, and <a href="javascript:toggleBio('prepr1')">[...]</a> 
                            <div id="prepr1" style="display: None">
                                <p>106,001 labels on all images. With this drastically expanded pediatric emotion centric database (>30x larger than existing public pediatric affect datasets), we trained a pediatric emotion classification convolutional neural network (CNN) classifier of happy, sad, surprised, fearful, angry, disgust, and neutral expressions in children. The classifier achieved 66.9% balanced accuracy and 67.4% F1-score on the entirety of CAFE as well as 79.1% balanced accuracy and 78.0% F1-score on CAFE Subset A, a subset containing at least 60% human agreement on emotions labels. This performance is at least 10% higher than all previously published classifiers, the best of which reached 56.% balanced accuracy even when combining "anger" and "disgust" into a single class. This work validates that mobile games designed for pediatric therapies can generate high volumes of domain-relevant datasets to train state of the art classifiers to perform tasks highly relevant to precision health efforts.</p>
                            </div>
                        
                        </p>
                    </div>
                </div>
                


        
   

        <script>
            function toggleBio(divID) {
              var moreAboutMe = document.getElementById(divID);
              if (moreAboutMe.style.display === "none") {
                moreAboutMe.style.display = "inline";
              } else {
                moreAboutMe.style.display = "none";
              }
            }
        </script>
    

        
            <div class="divider"></div>
    </body>
</html>
